protobuf/v1/pool.proto

```diff
@@ -36,6 +36,9 @@ message CreatePoolRequest {
   google.protobuf.StringValue uuid = 2; // optional uuid for the pool to be created
   repeated string disks = 3; // disk device paths or URIs to be claimed by the pool
   PoolType pooltype = 4;     // type of the pool
+  // Default cluster size is 4MiB (4 * 1024 * 1024), input cluster size must be multiple of 1MiB, 
+  // else default cluster size will be considered.
```

Comment:
An error must be returned in the case when a bad cluster size is specified, instead of silently ignoring incorrect size.
Also, add to the comment the unit of the size ("bytes" in this case).

Reply:
Done the changes.